### DISCUSSION TITLE
YimMenu: [CRASHING]

### DISCUSSION THREAD

**Discussion by mvcula** [YimMenu/YimMenu#2709] [General]
Hi, i have problem with menu.. general when i injects file yimmenu.dll with xenos my gta online crashing. I disabled all lua scripts and then this same thing happens.  Only without menu gta online working idk whats going on. Take screenshot last console logs. Sorry for my language i dont speak very well eng.

Greetings and thank you for help.
![Zrzut ekranu 2024-02-12 221851](https://github.com/YimMenu/YimMenu/assets/159834277/34f56008-d005-470c-a156-dde7d3ca062d)

**Comment by GreySideMM** [comment]
Please post the entire log as text file, not screenshot (where parts are missing).
You find the log in
C:/users/<YOUR-USER>/AppData/Roaming/YimMenu
and there the cout.log.

**Reply by mvcula** [reply]
[cout.log](https://github.com/YimMenu/YimMenu/files/14251147/cout.log)

**Comment by GreySideMM** [comment]
You still have a lot of Luas loaded and they produce a lot of errors. Move all files to the folder "disabled" in scripts. Create it if it doesn't exist. Or move them somewhere else.

**Reply by mvcula** [reply]
This same, i moved all to folder disabled :/
